banzai/images.py

```diff
@@ -172,6 +172,8 @@ def read_images(image_list, pipeline_context):
     for filename in image_list:
         try:
             image = Image(pipeline_context, filename=filename)
+            if image.telescope_id is None:
+                raise dbs.TelescopeMissingException
             munge(image, pipeline_context)
             if image.bpm is None:
                 bpm = image_utils.get_bpm(image, pipeline_context)
```

Comment:
Could this line be moved to `Image` class instantiator? It could help simplify the logic here.

Reply:
It would. But that totally defaults the purpose of this. The problem is that right now we can't create Image objects without a DB even for testing.

Reply:
Ah yes I had misunderstood how the BPM was set in the instantiator. At any rate I guess this section will be changed in the "override BPM" branch.